src/main/java/dk/aau/cs/verification/VerifyTAPN/VerifyTAPNExporter.java

```diff
@@ -61,9 +62,15 @@ public ExportedVerifyTAPNModel export(TimedArcPetriNet model, TAPNQuery query, F
 			PrintStream queryStream = new PrintStream(queryFile);
             if (query == null) {
                 throw new FileNotFoundException(null);
+            } else if (query.getCategory() == QueryCategory.CTL || (lens != null && !lens.isGame() && lens.isColored())) {
+                CTLQueryVisitor XMLVisitor = new CTLQueryVisitor();
+                queryStream.append(XMLVisitor.getXMLQueryFor(query.getProperty(), dataLayerQuery == null? model.name() : dataLayerQuery.getName(), false));
```

Comment:
Remove

Reply:
Fixed in https://github.com/TAPAAL/tapaal-gui/pull/39/commits/e0651b05f7e4c6329031295bd8540cab513b49f0